modular_skyrat/modules/gun_cargo/code/objects/box.dm

```diff
@@ -8,7 +8,7 @@
 
 /obj/item/storage/box/syringes/bluespace
 	name = "bluespace syringe box"
-	desc = "A three-pack of piercing syringes."
+	desc = "A three-pack of bluespace syringes."
```

Comment:
is this for syringe guns? I regret to inform you that bluespace syringes do not work in syringe guns.

Reply:
I'm aware, it's intended as a high-cost replacement for R&D